src/huggingface_hub/inference/_generated/types/chat_completion.py

```diff
@@ -45,17 +45,47 @@ class ChatCompletionInputMessage(BaseInferenceType):
     tool_calls: Optional[List[ChatCompletionInputToolCall]] = None
 
 
-ChatCompletionInputGrammarTypeType = Literal["json", "regex", "json_schema"]
+@dataclass_with_extra
+class JSONSchema(BaseInferenceType):
```

Comment:
```suggestion
class ChatCompletionInputJSONSchema(BaseInferenceType):
```

I'd make it more obvious what it is used for (especially because it's exported at the root of `huggingface_hub`)